wies/core/static/js/sidepanel.js

```diff
@@ -0,0 +1,34 @@
+// Simple sidepanel utilities for HTMX views
+(function() {
+  'use strict';
+  
+  // Initialize panel from URL on page load
+  document.addEventListener('DOMContentLoaded', function() {
+    const params = new URLSearchParams(window.location.search);
+    if (params.has('colleague') || params.has('assignment')) {
+      htmx.ajax('GET', window.location.href, {
+        target: '#sidepanel-container', 
+        swap: 'innerHTML'
+      });
+    }
+  });
+  
+  // Close panel function for templates
+  window.closeSidepanel = function() {
```

Comment:
i was surprised to see this function here and don't think it needs to exist:

the backend already constructs a `close_url` that the frontend can use to generate link element that the user will be directed to when clicking